packages/salmon-oracles-functions/src/index.ts

```diff
@@ -24,28 +24,44 @@ export class OraclesManager {
     })
   }
 
-  async updatePrices (oracleId: string, token: string, prices: TokenAmount[]): Promise<void> {
-    const transaction: TransactionSegWit = await this.builder.oracles.setOracleData({
+  /**
+   * Pushes prices to the price oracle on the blockchain.
+   *
+   * @param {string} oracleId
+   * @param {string} token
+   * @param {TokenAmount[]} prices
+   * @param {BigNumber} timestamp (optional)
+   * @return {Promise<void>}
+   */
+  async updatePrices (oracleId: string, token: string, prices: TokenAmount[], timestamp?: BigNumber): Promise<void> {
```

Comment:
```suggestion
  async updatePrices (
    oracleId: string, 
    token: string, 
    prices: TokenAmount[], 
    timestamp: BigNumber = new BigNumber(Math.floor(Date.now() / 1000)
  ): Promise<void> {
```

This could work, and maybe better for readbility?